### ISSUE TITLE
ArgoUML cannot be installed on any windows systems

### ISSUE THREAD

**Issue by vishnumur777** [argouml-tigris-org/argouml#61]
ArgoUML cannot be installed be installed on any windows based systems as it can't detect JRE path and it have to be download from sun.com to download JRE even if it is installed . when I disable the JRE checkbox it cannot be installed without JRE even if it is there on the system. I have checked for windows 11,10 and windows 8.

**Comment by linustolke** [comment]
Yes, well the work with this is very slow. I have set up tests to build with openjdk 8, 11 and 17 and that succeeds, at least in a linux container on Linux. This is mostly to verify where we stand in this. I hope that someone will start working with this.
